rotkehlchen/tests/api/test_eth2.py

```diff
@@ -759,6 +764,16 @@ def test_query_combined_mev_reward_and_block_production_events(rotkehlchen_api_s
             'eth2validatorsresource',
         ), json={'validator_index': vindex1},
     )
+    response = requests.put(
+        url=api_url_for(
+            rotkehlchen_api_server,
+            'eth2validatorsresource',
+        ), json={'validator_index': vindex2},
+    )
```

Comment:
the way you moved the code you are not checking neither this nor the previous API call for correctness, while it was being checked before

```suggestion
    )
    assert_simple_ok_response(response)
```

Reply:
I was only checking the second but there is also a check to assert that the expected number of validators are in the db